<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FilterSearchWidget</class>
 <widget class="QWidget" name="FilterSearchWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>635</width>
    <height>32</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Filter/Search:</string>
     </property>
     <property name="buddy">
      <cstring>leFilterText</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="leFilterText">
     <property name="minimumSize">
      <size>
       <width>180</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="cboxFilterType"/>
   </item>
   <item>
    <widget class="QCheckBox" name="cbAutomatic">
     <property name="text">
      <string>Automatic</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="cbCaseSensitive">
     <property name="text">
      <string>Match case</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QToolButton" name="tbCloseWidget">
     <property name="text">
      <string notr="true"/>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
